hibike/devices/LimitSwitch/limit_switch.cpp

```diff
@@ -45,7 +45,7 @@ uint8_t device_read(uint8_t param, uint8_t* data, size_t len) {
   if (MAX_PAYLOAD_SIZE - len < sizeof(uint8_t) || param >= NUM_SWITCHES || param<0) {
     return 0;
   }
-  data[0] = digitalRead(pins[param]) == LOW;
+  data[0] = digitalRead(pins[param]);
```

Comment:
Note that making the param an int will impact the way it's presented to the students. It has generally been our policy to convert params to accessible units in firmware (i.e. analog sensors are returned as floats and not integers from 0 to 1024). So I wouldn't recommend making it an int.